packages/design-picker/src/components/featured-picks-buttons/index.tsx

```diff
@@ -28,10 +29,13 @@ const FeaturedPicksButtons: React.FC< Props > = ( { className, designs, onSelect
 					isSecondary
 					onClick={ () => onSelect( design ) }
 				>
-					{
+					{ sprintf(
 						/* translators: %s is the title of design */
-						sprintf( __( 'Use %s' ), design.title )
-					}
+						__( 'Use %s' ),
+						isBlankCanvasDesign( design )
+							? __( 'Blank Canvas', __i18n_text_domain__ )
+							: design.title
+					) }
```

Comment:
In the interests of making the design picker the same everywhere, maybe we should just change `blank-canvas`'s title in `available-designs-config.json` to "Blank Canvas".

Reply:
I want to avoid any side effects when we turn off this feature. But it seems we replace the title with “Blank Canvas” everywhere, so maybe we can just rename the title directly 👍